src/main/java/seedu/address/logic/commands/deletestudentcommands/DeleteStudentByEmailCommand.java

```diff
@@ -61,4 +69,15 @@ public String toString() {
                 .add("email", email)
                 .toString();
     }
+
```

Comment:
Can this method be put in the parent class so that it does not have to be repeated in every sub class of DeleteStudentCommand? This is to minimise code duplication